lib/fathead/mongoosejs/output.txt

```diff
@@ -0,0 +1,640 @@
+Mongoose-Aggregate	A			Mongoose						http://mongodb-tools.com/img/mongoose.png	<section class="prog__container"><pre><code>Mongoose#Aggregate()</code></pre><p>The Mongoose Aggregate constructor</p></section>	http://mongoosejs.com/docs/api.html#index_Mongoose-Aggregate
```

Comment:
`Mongoose` is too generic -- is there a better name we can use here? Something more specific?

We should prefix the other category names with "Mongoose" as well I think, because they're quite generic terms (e.g "Connection")